### PR TITLE
[FIX] Polynomial Regression: no continuous variables

### PR DESCRIPTION
**Pull request by jerneju**
#### Issue
Widget crashes when no cont. variables.

### CODE REVIEW

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3-educational/issues/309224174/

File: orangecontrib/educational/widgets/owpolynomialregression.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ class Error(OWWidget.Error):
         """
         all_none = Msg("One of the features has no defined values")
 
+    class Warning(OWWidget.Warning):
+        no_cont_variables = Msg("No continuous variables")
```
Comment (kernc) [inline orangecontrib/educational/widgets/owpolynomialregression.py]: This looks like an Error, if it breaks execution. Might also reword into something nicer.

Comment (jerneju) [reply orangecontrib/educational/widgets/owpolynomialregression.py]: Done.

File: orangecontrib/educational/widgets/owpolynomialregression.py [hunk 2]
```diff
@@ -63,6 +63,7 @@ class Error(OWWidget.Error):
         Class used fro widget warnings.
         """
         all_none = Msg("One of the features has no defined values")
+        no_cont_variables = Msg("Require numeric features")
```
Comment (kernc) [inline orangecontrib/educational/widgets/owpolynomialregression.py]: Verb, object. Where the subject? Require user-friendly messages.

File: orangecontrib/educational/widgets/owpolynomialregression.py [hunk 3]
```diff
@@ -177,22 +178,27 @@ def clear_plot(self):
...
             cvars = [var for var in data.domain.variables if var.is_continuous]
             class_cvars = [var for var in data.domain.class_vars
                            if var.is_continuous]
 
-            self.x_var_model[:] = cvars
-            self.y_var_model[:] = cvars
-
             nvars = len(cvars)
             nclass = len(class_cvars)
+            if nvars == 0:
```
Comment (PrimozGodec) [inline orangecontrib/educational/widgets/owpolynomialregression.py]: When no numeric features `self.x_var_model[:]` and `self.y_var_model[:]§` should be set to `[]`. 
Moving entire `if` clause two lines lower would solve a problem.

File: orangecontrib/educational/widgets/owpolynomialregression.py [hunk 4]
```diff
@@ -63,6 +63,7 @@ class Error(OWWidget.Error):
         Class used fro widget warnings.
         """
         all_none = Msg("One of the features has no defined values")
+        no_cont_variables = Msg("Polynomial Regression requires at least one numerical feature.")
```
Comment (PrimozGodec) [inline orangecontrib/educational/widgets/owpolynomialregression.py]: I wold replace word `numerical` with `numeric`.

**Pull request merged**